mypy_django_plugin/main.py

```diff
@@ -60,6 +62,30 @@ def add_new_manager_base_hook(ctx: ClassDefContext) -> None:
     helpers.add_new_manager_base(ctx.api, ctx.cls.fullname)
 
 
+def reparametrize_manager_base_hook(ctx: ClassDefContext) -> None:
```

Comment:
Have we tested reparametrization works when inheriting

- `django.db.models.Manager`, and
- `django.db.models.manager.BaseManager`

?